bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZPrimeToTT/ZPrimeToTT_M500GeV_W50GeV/ZPrimeToTT_M500GeV_W50GeV_proc_card.dat

```diff
@@ -0,0 +1,8 @@
+import model topBSM_UFO-ckm_no_b_mass
+define p = g u c d s b u~ c~ d~ s~ b~ 
+define j = g u c d s b u~ c~ d~ s~ b~
+generate p p > s1 > t t~ , (t > W+ b , W+ > l+ vl ), (t~ > W- b~ , W- > j j ) 
```

Comment:
You still need to define l+ = e+ mu+ ta+, assuming you also like to consider taus?